src/webrtc/call.ts

```diff
@@ -1370,6 +1399,28 @@ export class MatrixCall extends EventEmitter {
         }
     }
 
+    async transfer(targetUserId: string, targetRoomId?: string) {
+        // Fetch the target user's global profile info: their room avatar / displayname
+        // could be different in whatever room we shae with them.
+        const profileInfo = await this.client.getProfileInfo(targetUserId);
+
+        const replacementId = genCallID();
+
+        const body = {
+            replacement_id: genCallID(),
+            target_user: {
+                id: targetUserId,
+                display_name: profileInfo.display_name,
+                avatar_url: profileInfo.avatar_url,
+            },
+            create_call: replacementId,
+        } as any;
```

Comment:
Hmm, seems like we're cheating a bit to mark these as `any`... Is it possible to create an interface with the possible fields?